src/site/content/en/blog/stories-component/index.md

```diff
@@ -65,14 +94,41 @@ More specifically, to me that meant:
 }
 ```
 
-{% CompareCaption %}
-Parent with overscroll defines snap behavior
-{% endCompareCaption %}
+Let's breakdown that `grid` value:
 
-{% endCompare %}
+* `1fr` lays out the row, and `auto-flow 100vw` lays out the column
+* [`1fr`](https://alligator.io/css/css-grid-layout-fr-unit/) is
+  technically "1 part of the available space" but in this case you can
+  think of it as filling the height 
+* The placement of `auto-flow` after the `/` is shorthand for 
+  [`grid-auto-flow: column`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow)
+* `100vw` sets the column to the full width of the device
```

Comment:
You already started to breakdown this `grid` shorthand but I found it pretty daunting so I tried to just break it down further and make it easier to grok